swig/cpp/src/Libyang.cpp

```diff
@@ -206,6 +206,31 @@ S_Data_Node Context::parse_fd(int fd, LYD_FORMAT format, int options) {
     S_Deleter new_deleter = std::make_shared<Deleter>(new_node, deleter);
     return std::make_shared<Data_Node>(new_node, new_deleter);
 }
+
+S_Module Context::parse_module_mem(const char *data, LYS_INFORMAT format) {
```

Comment:
please unify naming - `lys_parse_path()` was wrapped as `Context::parse_path()`, so similarly here I would expect `Context::parse_fd()` and `Context::parse_mem()`

Reply:
It was not possible to ta name it that because it would get in conflict with:
`S_Data_Node parse_mem(const char *data, LYD_FORMAT format, int options = 0);`

I can name it `parse_mem()` if I remove the default value `options = 0` from `S_Data_Node parse_mem()`.

What would you like to do?

Reply:
Then, please, rename `parse_path()` to `parse_module_path()` and probably it would be also better to have `parse_data_*()` functions. Currently we mix wrapper names for lys_ and lyd_ functions.